<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectParkingLotDialog</class>
 <widget class="QDialog" name="SelectParkingLotDialog">
  <property name="minimumSize">
   <size>
    <width>520</width>
    <height>420</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text"><string>请选择停车场与车辆</string></property>
     <property name="alignment"><set>Qt::AlignCenter</set></property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="table_lots">
     <property name="columnCount"><number>3</number></property>
     <column>
      <property name="text"><string>停车场ID</string></property>
     </column>
     <column>
      <property name="text"><string>名称</string></property>
     </column>
     <column>
      <property name="text"><string>地址</string></property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayout">
     <item>
      <widget class="QComboBox" name="combo_vehicles"/>
     </item>
     <item>
      <spacer name="hSpacer"/>
     </item>
     <item>
      <widget class="QPushButton" name="btn_ok">
       <property name="text"><string>确定停车</string></property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="text"><string>取消</string></property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
